src/sage/plot/graphics.py

```diff
@@ -429,7 +429,16 @@ def set_legend_options(self, **kwds):
 
         ::
 
-            sage: p.set_legend_options(loc=(0.5,0.5)); p  # aligns the bottom of the box to the center                  # needs sage.symbolic
+            sage: p.set_legend_options(loc=(0.5,0.5)); p  # aligns the bottom of the box to the center                # needs sage.symbolic
+            Graphics object consisting of 1 graphics primitive
+
+        The parameters ``loc`` and ``borderaxespad`` can be altered in order to place the legend below the x-axis label or to the left of the y-axis label.
+        The following example sets ``loc=3`` and ``borderaxespad = -7.5-0.01*p.fontsize()`` so as to obtain the legend below the x-axis label::
```

Comment:
```suggestion
        The parameters ``loc`` and ``borderaxespad`` can be altered
        in order to place the legend below the x-axis label or to
        the left of the y-axis label::
```
We try to keep 80 char/line for docstrings in general. The second sentence is redundant with the example presented.